flight/Modules/Actuator/actuator.c

```diff
@@ -66,38 +67,35 @@
 
 // Private types
 
-
 // Private variables
 static struct pios_queue *queue;
 static struct pios_thread *taskHandle;
 
-// used to inform the actuator thread that actuator update rate is changed
-static volatile bool actuator_settings_updated;
-// used to inform the actuator thread that mixer settings are changed
-static volatile bool mixer_settings_updated;
+// used to inform the actuator thread that actuator / mixer settings are updated
+static volatile bool settings_updated;
+
+// The actual mixer settings data, pulled at the top of the actuator thread
+MixerSettingsData mixerSettings;
+
+// Ditto, for the actuator settings.
+ActuatorSettingsData actuatorSettings;
 
 // Private functions
 static void actuatorTask(void* parameters);
-static float scaleChannel(float value, float max, float min, float neutral);
-static void setFailsafe(const ActuatorSettingsData * actuatorSettings, const MixerSettingsData * mixerSettings);
+static float scaleChannel(float value, int idx);
```

Comment:
I'll go ahead and rename functions (ignoring variables because that's just too much diff).  Every single possible convention is currently used and it's yucky.